www_src/pages/project/project.jsx

```diff
@@ -53,7 +54,8 @@ var Project = React.createClass({
         x: 0,
         y: 0
       },
-      zoom: DEFAULT_ZOOM
+      zoom: DEFAULT_ZOOM,
+      isPageZoomed: false
```

Comment:
It's not really the same thing though. Zooming in on a page isn't "selecting" it.